SCons/Tool/MSCommon/common.py

```diff
@@ -106,7 +106,8 @@ def read_script_env_cache():
         try:
             p = Path(CONFIG_CACHE)
             with p.open('r') as f:
-                envcache = json.load(f)
+                envcache_list = json.load(f)
```

Comment:
This seems fine, I'm inclined to merge this change.  Maybe a comment why we're doing what we're doing with came out of the json file (and/or what goes into it).

Reply:
Better or worse?  See below as well.